lib/src/presentations/job_position/cubit/job_position_state.dart

```diff
@@ -0,0 +1,18 @@
+part of 'job_position_cubit.dart';
+
+class JobPositionState extends Equatable {
+  const JobPositionState({this.dataState, required this.text});
+
+  final DataState<List<String>>? dataState;
+  final String text;
+
+  JobPositionState copyWith({
+    DataState<List<String>>? dataState,
+    String? text,
+  }) {
+    return JobPositionState(dataState: dataState, text: text ?? this.text);
+  }
+
```

Comment:
Should be define factory class for set default value in cubit.
Example: 
```
factory JobPositionState.ds(){
return JobPositionState(text: "");
}
```

Reply:
I fixed it